lambdas/utils/lloyd_george_validator.py

```diff
@@ -131,32 +123,14 @@ def check_for_file_names_agrees_with_each_other(file_name_list: list[str]):
         raise LGInvalidFilesException("File names does not match with each other")
 
 
-def validate_with_pds_service(file_name_list: list[str], nhs_number: str):
+def validate_with_pds_service(
```

Comment:
would it be good to rename this method to something like `validate_filenames_with_patient_info`?
Because now the functionality of fetching PDS API is moved elsewhere